src/RVMaster/rvmonitor.rv

```diff
@@ -0,0 +1,14 @@
+#include <cstring>
+#include <string>
+#include "ros/ros.h"
+
+testSpecOne() {
+    int a = 10;
+    int b;
+
+    event controlMessageOne(std::string msg) /chatter std_msgs/String '{data:msg}'
+    {
+        msg = msg + "RV";
+        ROS_INFO("%s", msg.c_str());
+    }
+}
```

Comment:
Should this file be placed under some sort of `test` folder to indicate it's only meant to be used as a test spec?